openpype/settings/lib.py

```diff
@@ -941,6 +941,35 @@ def get_general_environments():
     return environments
 
 
+def _list_path_items(folder_structure):
+    output = []
+    for key, value in folder_structure.items():
+        if not value:
+            output.append(key)
+        else:
+            paths = _list_path_items(value)
+            for path in paths:
+                if not isinstance(path, (list, tuple)):
+                    path = [path]
+
+                output.append([key, *path])
+
+    return output
+
+
+def get_project_basic_paths(project_name):
```

Comment:
These functions have nothing to do with settings except they're using them. Move them to lib please.

Reply:
Moved to lib/path_tools.py